websites/manopozicija.lt/ansible.cfg

```diff
@@ -1,6 +1,9 @@
 [defaults]
 inventory = inventory.cfg
```

Comment:
Here should be `hostfile`?

Reply:
No? https://docs.ansible.com/ansible/latest/reference_appendices/config.html#default-host-list:

Section: `[defaults]`, key: `inventory`.